CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # dbt_Recharge_source v0.1.0
 🎉 This is the initial release of this package! 🎉
-# 📣 What does this dbt package do?
+# 📣 What does this dbt package do? 
```

Comment:
Can you provide a link at the end of this CHANGELOG entry to point to the README. Similar to what we did with [Reddit Ads](https://github.com/fivetran/dbt_reddit_ads/blob/main/CHANGELOG.md).